include/maxscale/dcb.h

```diff
@@ -176,11 +178,13 @@ typedef struct dcb
     bool            ssl_write_want_read;    /*< Flag */
     bool            ssl_write_want_write;    /*< Flag */
     bool            was_persistent;  /**< Whether this DCB was in the persistent pool */
+    bool            high_water_has_reached; /** High water mark has reached, to determine whether need release throttle */
```

Comment:
Minor thing, but `high_water_reached` would be better, as `high_water_has_reached` is not grammatically correct. `high_water_has_been_reached` would be, but that is unnecessarily long.